client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependecies
+ */
+import shortcodeUtils from 'lib/shortcode';
+import { deserialize } from 'components/tinymce/plugins/simple-payments/shortcode-utils';
+import { getSimplePayments } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import formatCurrency from 'lib/format-currency';
+import QuerySimplePayments from 'components/data/query-simple-payments';
+
+class SimplePaymentsView extends Component {
```

Comment:
Maybe you are right but working in tinymce brings a lot of funky stuff. For example, the styling must be loaded inside the tinymce iframe and not outside, otherwise it won't be applied. That's why I wanted to keep it here (+ also other previews have it here).